timemachine/fe/mcgregor.py

```diff
@@ -411,6 +421,35 @@ def atom_map_pop(map_1_to_2, map_2_to_1, idx, jdx):
     map_2_to_1[jdx] = UNMAPPED
 
 
+def is_chiral_conversion(bond_graph: nx.Graph, disabled_bonds: Set[Tuple[int, int]]) -> bool:
```

Comment:
(actually not sure of the best terminology here ... probably fine as is)

Reply:
Still might want to revisit some of the naming here, e.g.  the flag `disallow_chiral_conversion`. This feels complicated enough that it's worth coming up with a good name if that can help make it any clearer.

Reply:
Renamed `disallow_chiral_conversion` to `enforce_chirally_valid_dummy_groups` in https://github.com/proteneer/timemachine/pull/1356/commits/e6b29ebf892b06c9d8440d1c18d5bf289322f5a7, since this seemed more accurate and less confusing.